test/functional/p2p_filter.py

```diff
@@ -104,6 +105,18 @@ def run_test(self):
             txid = self.nodes[0].sendtoaddress(self.nodes[0].getnewaddress(), 7)
             filter_node.wait_for_tx(txid)
 
+        self.log.info('Check that request for filtered blocks is ignored if no filter is set')
+        filter_node.merkleblock_received = False
+        self.nodes[0].generatetoaddress(1, self.nodes[0].getnewaddress())
+        filter_node.sync_with_ping()
+        filter_node.sync_with_ping()
```

Comment:
I see Marco's "wait enough pings" approach is becoming a pattern now.. ;) .  Might I suggest wrapping this up into a nicely-named method?  Not only will this help future contributors understand intent but if anyone ever comes up with a better approach, they can easily replace all the right instances of multiple `sync_with_pings`.

Reply:
Hi chanhosuh, thanks for reviewing and the suggestion! Seems like you have followed the discussion in PR #18334 where this pattern first appeared :) I like the idea of a method, but don't have an idea of a good descriptive name. I anyways think it would be too much stuffing this also in this PR. May you want to come up with a follow-up PR?

Reply:
I don't have an idea for a good name either :).  Anyway, the recent changes have seemingly deprecated this topic.